frontend/src/app/_models/globals.ts

```diff
@@ -0,0 +1,88 @@
+import { Injectable } from "@angular/core";
+import notify from 'devextreme/ui/notify';
+import { DatePipe } from "@angular/common";
+
+
+@Injectable()
+export class Globals {
+
+  public toastTimeDuration: number = 1000;
+  public toastSuccessType: string = "success";
+  public toastErrorType: string = "error";
+  public companyName: string = "S4";
+  public imageProfileURLDir: string = 'https://portal.s4.co.za/gallery/images/staff/';
```

Comment:
I know this is a demo so its fine. But this could rather be separated into a host and then referenced in a new variable for the actual images